jupyter-book/introduction/interoperability.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<p>Lots of numba warnings here. Can we filter them, please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/single-cell-best-practices/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<ul><li>Probably, if you tell me how 😸</li><li>Maybe it's better to leave them so users aren't surprised when they see them?</li><li>Is there some way to avoid them maybe?</li></ul>

Reply:
<p><a href="https://github.com/lmcinnes/umap/issues/1004" target="_blank">https://github.com/lmcinnes/umap/issues/1004</a></p><p> </p><p>update your UMAP package :)</p>